java-frontend/src/test/files/se/LocksNotUnlockedCheck.java

```diff
@@ -189,7 +190,7 @@ public void while_statement() {
       lock.tryLock();
     }
     lock.unlock();
-    while (foo) {
+    while (Random.nextBoolean()) {
```

Comment:
This was temporary and I forgot to put test back to original state after fixing the problem with flow paths. Thanks for catching it.